### DISCUSSION TITLE
Error while trying to run a Logic App in Visual Studio Code: Failed to find dependency "windows-process-tree"

### DISCUSSION THREAD

**Discussion by FerryMan-Coder** [Azure/logicapps#752] [Q&A] [answered]
I'm trying to get the Azure Logic App sample on https://learn.microsoft.com/en-us/azure/logic-apps/create-single-tenant-workflows-visual-studio-code to work.
I'm working with the following versions
Visual Studio Code, Version: 1.77.0 (user setup)
Azure Logic Apps (Standard), 2.0.16

When I try to run the code, I get: Failed to find dependency "windows-process-tree". Maybe I'm not doing the right searches, but I can't seem to find where this error is coming from. Anyone any ideas?

Cheers,
FerryMan

**Comment by sandokanfirst** [comment, marked as answer]
I tried the same approach at both my home and my work PC, with 2 differences. On my work PC I didn't install the latest Node.js version, but v14.21.3. On both PCs I executed npm install windows-process-tree, but on my work PC I executed it in the directory my VS Code project was in and I initialized with npm cache clean --force. I don't know what did the trick, but my Logic App now works from within VS Code.

![testtest](https://user-images.githubusercontent.com/36963445/229822677-2ec3e033-817e-4abe-b734-25314fcdcb27.jpg)

Edit: I now think that I wasn't able to build "windows-process-tree" because I failed to choose the option to install "the necessary tools" when installing and setting up Node.js!
